client/src/locales/en-US.json

```
@@ -551,24 +551,24 @@
                 "cancelGreet": "Cancel"
             },
             "subtitles": {
-                "provideHostCode": "Click the code below on the guest.",
-                "getGuestCode": "Click on the code that appears on the guest.",
+                "provideHostCode": "Click the code below on the guest device.",
+                "getGuestCode": "Click on the code that appears on the guest device.",
                 "summary": "The device {label} has been added to your profile.",
-                "getDeviceInfo": "Waiting for you to fill the information on the guest.",
-                "waitForGuest": "You need to have Parsec open on both devices for the inscription duration.",
+                "getDeviceInfo": "Waiting for you to fill the information on the guest device.",
+                "waitForGuest": "You must have Parsec open on both devices for the inscription duration.",
                 "scanOrShare1": "Scan the QR Code ",
                 "scanOrShare2": "or ",
                 "scanOrShare3": "share the link",
                 "cancelGreet": "Are you sure you want to cancel the greeting process?",
                 "copiedToClipboard": "Copied to clipboard"
             },
             "errors": {
-                "retrieveDeviceInfoFailed": "Could not retrieve the information.",
+                "retrieveDeviceInfoFailed": "Failed to retrieve the information.",
```

If this is related to device information, maybe it is better to be specific:
```suggestion
                "retrieveDeviceInfoFailed": "Failed to retrieve device information.",
```